.changeset/hot-dogs-bake.md

```diff
@@ -0,0 +1,18 @@
+---
+"@logto/browser-sample": patch
```

Comment:
nit: maybe `minor`? To me it's quite a big change actually, and it might causing potential issues to existing projects

Reply:
major then, since we changed the modules type from CJS to ESM

Reply:
Yeah, OK